### DISCUSSION TITLE
Custom Button ColorScheme.

### DISCUSSION THREAD

**Discussion by SirReaDy** [chakra-ui/chakra-ui#4883] [General]
Hi everyone.
I have a feature on my website where the user can pick the main color of the website.
The color is saved in the const `linkColor.`

Here is the example that does NOT work.
```javascript
<Button
              as="a"
              variant="solid"
 colorScheme={`${linkColor}`}
OR
 colorScheme={linkColor}

              display="inline-flex"
              alignItems="center"
              justifyContent="center"
              w={{ base: "full", sm: "auto" }}
              mb={{ base: 2, sm: 0 }}
              size="lg"
              cursor="pointer"
            >
```

How can i achieve what i need.
For texts it works

```javascript
          <Text
              bgGradient={`linear(to-r, ${linkColor},purple.500)`}
            >
           ```

Best Regards.

**Comment by primos63** [comment]
`colorScheme` requires a color defined in the theme which has levels 50-900. The component will append the level to the color. That's how it adjusts for light and dark modes and how it determines accent colors like border, hover, etc. 

It works for `Text` because linear gradient only needs a color value. Here you specified `${linkColor}` which I assume is a hex value. `purple.500` also resolves to a hex value.

I added this case to my [CodeSandbox](https://codesandbox.io/s/chakra-ui-theme-extension-w5u2n?file=/src/sections/ButtonSection.js) of theme examples.
